R/robustbayesianmetaanalysis.R

```diff
@@ -106,195 +106,195 @@ RobustBayesianMetaAnalysis <- function(jaspResults, dataset, options, state = NU
 
 .robmaDependencies <- c(
   "measures",
-  "fitted_path",
-  "cohensd_testType",
-  "input_ES",
-  "input_t",
-  "input_SE",
-  "input_CI",
-  "input_N",
-  "input_N1",
-  "input_N2",
-  "input_labels",
-  "effect_direction",
-  "priors_mu",
-  "priors_tau",
-  "priors_omega",
-  "priors_mu_null",
-  "priors_tau_null",
-  "priors_omega_null",
-  "advanced_control",
-  "advanced_omit_prior",
-  "advanced_omit_marglik",
-  "advanced_omit_theta",
-  "advanced_omit_error",
-  "advanced_omit_error_value",
-  "advanced_omit_ESS_value",
-  "advanced_omit_ESS",
-  "advanced_omit_rhat_value",
-  "advanced_omit_rhat",
-  "advanced_omit",
-  "advanced_autofit_error",
-  "advanced_autofit_rhat",
-  "advanced_autofit_time_unit",
-  "advanced_autofit_time",
-  "advanced_autofit",
-  "advanced_thin",
-  "advanced_chains",
-  "advanced_iteration",
-  "advanced_burnin",
-  "advanced_adapt",
-  "advanced_bridge_iter",
-  "advanced_mu_transform"
+  "fittedPath",
+  "cohensDTestType",
+  "inputES",
+  "inputT",
+  "inputSE",
+  "inputCI",
+  "inputN",
+  "inputN1",
+  "inputN2",
+  "inputLabels",
+  "effectDirection",
+  "priorsMu",
+  "priorsTau",
+  "priorsOmega",
+  "priorsMuNull",
+  "priorsTauNull",
+  "priorsOmegaNull",
+  "advancedControl",
+  "advancedOmitPrior",
+  "advancedOmitMarglik",
+  "advancedOmitTheta",
+  "advancedOmitError",
+  "advancedOmitErrorValue",
+  "advancedOmitESSValue",
+  "advancedOmitESS",
+  "advancedOmitRhatValue",
+  "advancedOmitRhat",
+  "advancedOmit",
+  "advancedAutofitError",
+  "advancedAutofitRhat",
+  "advancedAutofitTimeUnit",
+  "advancedAutofitTime",
+  "advancedAutofit",
+  "advancedThin",
+  "advancedChains",
+  "advancedIteration",
+  "advancedBurnin",
+  "advancedAdapt",
+  "advancedBridgeIter",
+  "advancedMuTransform"
 )
 # priors related functions
-.robmaOptions2Priors      <- function(options_prior) {
-  options_prior <- .robmaOptions2PriorsEval(options_prior)
+.robmaOptions2Priors      <- function(optionsPrior) {
+  optionsPrior <- .robmaOptions2PriorsEval(optionsPrior)
```

Comment:
It's not fully clear to me what this function does. From the name I'd guess that it takes in optionsPrior and returns a prior. It's unclear to me what optionsPrior is and I'm also unsure about the output. Is that a function? Is that a list? Is that a set of points on the yDomain? Furthermore, I'd reckon that the renaming of optionsPrior can be a bit ambiguous. Perhaps this relates to me not understanding what .robmaOptions2PriorsEval does.

Reply:
The `options` generated by qml contain a list of priors for different parameters, this function is used to convert each of the priors specified in qml and convert it to a `RoBMA::prior` object, that can be further visualized, or passed into the fitting function. The `.robmaOptions2PriorsEval` is used for evaluating qml arguments that are passed as R arguments, `1/2`, `pi`, and etc...

Reply:
It would be good to some how indicate this in the function itself.

Reply:
renamed